app/gui/view/graph-editor/src/lib.rs

```diff
@@ -3536,7 +3535,7 @@ fn new_graph_editor(app: &Application) -> GraphEditor {
     //     - If it was long, disable vis which were disabled (preview mode).
 
     let viz_press_ev      = inputs.press_visualization_visibility.clone_ref();
-    let viz_d_press_ev    = inputs.double_press_visualization_visibility.clone_ref();
+    let viz_open_fs_ev = inputs.open_fullscreen_visualization.clone_ref();
```

Comment:
Missing aligning spaces. It's not our convention to use them everywhere, but let's maintain consistency with the rest of the block.

Reply:
I had comments on other PRs of mine where I tried to maintain consistency and was told to follow the style guide. I don't like it either though. What Adam suggests might be the best compromise.